scio-repl/src/main/scala/com/spotify/scio/repl/ScioILoop.scala

```diff
@@ -227,13 +227,22 @@ class ScioILoop(scioClassLoader: ScioReplClassLoader,
     }
   }
 
+  private def loadIoCommands(): IR.Result = {
+    intp.interpret(
+      """
+        |val _ioCommands = new com.spotify.scio.repl.IoCommands(sc.options)
+        |import _ioCommands._
+      """.stripMargin)
+  }
+
   override def createInterpreter(): Unit = {
     super.createInterpreter()
     welcome()
     intp.beQuietDuring {
       addImports()
       createBigQueryClient()
       newScioCmdImpl("sc")
+      loadIoCommands()
```

Comment:
I wonder if we should think about doing some of this stuff async to save on startup time?